src/test/lib/scheduler/operator_task_test.cpp

```diff
@@ -98,23 +104,22 @@ TEST_F(OperatorTaskTest, MakeDiamondShape) {
 
   const auto& [tasks, root_operator_task] = OperatorTask::make_tasks_from_operator(union_positions);
 
-  ASSERT_EQ(tasks.size(), 5u);
+  ASSERT_EQ(tasks.size(), 5);
```

Comment:
Not yours, but we can actually use `EXPECT` here because we build a set and do not access elements in the vector directly, right? So no segfault that would be prevented and a single execution would reveal which task is missing